tests/lib/devicetree/emul/src/main.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Google Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device_extern.h>
```

Comment:
Do we need to include this? I thought this is automatically included in every file by the build system.